backend/.template-env

```diff
@@ -16,6 +16,9 @@ DJANGO_ADMIN_PATH=admin/
 # Hostname
 DJANGO_HOST=localhost
 
+# Extra host value for ALLOWED_HOSTS (TODO: replace)
```

Comment:
Why is there a TODO?

Reply:
this PR is about getting email login to work
Would it be more appropriate to keep this workaround and come back to this in a different PR or better to refactor it here and now?

Reply:
fixed